### PR TITLE
Locale ja jp

### PR DESCRIPTION
**Pull request by Narita-1095305**
add ja_JP locale

### CODE REVIEW

**Review by maehne — CHANGES_REQUESTED** [review]
Thanks a lot @Narita-1095305 for your work on the Japanes localization! Some keys need to be changed due to recently merged PR #480. 

Additionally, you seem to have copied the entire English documentation into a dedicated Japanes doc directory without any modification. As long as you don't plan to translate all these files to Japanese, this is not necessary, as Logisim automatically falls back to display the English help if a translated version is missing. Therefore, I would propose to leave out these files from this PR. Or did I maybe miss something?

It would be great if you could carry out these changes and optionally squash the commits so that the copies of the unmodified English documentation files get left out. Thanks!

**Comment by Narita-1095305** [comment]
Finished some keyword changes and remove unmodified English documentation files.

File: src/main/resources/resources/logisim/settings.properties [hunk 1]
```diff
@@ -1 +1 @@
-locales = de el en es fr pt ru it nl pl
+locales = de el en es fr pt ru it nl pl ja_JP
```
Comment (maehne) [inline src/main/resources/resources/logisim/settings.properties:1]: I guess `ja_JP` should be changed to `ja` to match the pattern of the other languages.

**Review by maehne — CHANGES_REQUESTED** [review]
Thanks @Narita-1095305 for taking into account my feedback so quickly! I think a small change in the `settings.properties` file is still necessary (see my related comment).

@BFH-ktt1: I would suggest to [squash and merge this PR](https://gitbetter.substack.com/p/how-to-squash-git-commits) to reduce the amount of changes to be merged.

**Comment by Narita-1095305** [comment]
I finished change ja_JP  to ja

**Comment by maehne** [comment]
This PR should be merged before PR #484 by @zdimension.

**Comment by maehne** [comment]
@Narita-1095305: Thanks a lot for all your work! @BFH-ktt1 just merged your PR. For the future, please do all pull requests on the `develop` branch.

**Comment by davidhutchens** [comment]
@BFH-ktt1, did you mean to merge this into master?

**Comment by BFH-ktt1** [comment]
@davidhutchens , no, I just did not pay attention

**Pull request merged**